ydb/docs/en/core/concepts/cdc.md

```diff
@@ -38,6 +38,26 @@ By default, virtual timestamps are not uploaded to the changefeed. To enable the
 
 {% endnote %}
 
+## Getting virtual timstamps during query execution
```

Comment:
The whole idea of returning virtual timestamps is not about CDC, sorry. You can use it without CDC. CDC is just a good example of this technology.
I can't find a good proper place for this information... But https://ydb.tech/docs/en/concepts/transactions (actually, YDB transactions and _queries_) looks like a better place to introduce virtual timestamps and to return them on query response